### DISCUSSION TITLE
How to run a different renovate.json?

### DISCUSSION THREAD

**Discussion by calto001** [renovatebot/renovate#29733] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Gitlab

### Please tell us more about your question or problem

For this I have one repo. In the root I have `renovate.json` which I use to handle my main updates. However I also use renovate to propagate changes for our internally developed node modules which I have `renovate2.json`. My goal is that when we push a new node module renovate triggers and runs the `renovate2.json` only. The reason I need to separate the files is because in the main `renovate.json` I have to explicitly use `includePaths` for the updates. But for our `renovate2.json` I dont  want to exclude any paths. 

Ive been trying various things but have been able to achieve my desired outcome. If any one has any ideas I would love to hear about them. Thank you for your help!

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
It's not possible, however you could run with requireConfig=ignored to fully ignore the renovate.json in the repo. Then you need to make sure the config for that run is fully in your config.js

**Reply by calto001** [reply]
Thats makes sense. I've been trying to use the `inheritConfig*` configuration options to pull in that new config.js from a different repo. But Renovate doesnt seem to be pulling it in. Im running Renovate and passing in those `inheritConfig*` as env variables. In the debug log it shows them in `Env config` but the logs show nothing for that new config.js. This repo is the same endpoint and also confirmed the token can access that repo. 

I was assuming that in the logs the combined config would show the env config and the config of the config being inherited. But it just shows the env config settings.

**Reply by calto001** [reply]
The only way I could get it to read that other config.js in a different repo was to do this inside the renovate job - Curl the config and save it. Then set `RENOVATE_CONFIG_FILE` to that config.js that was curl'd.
